src/quicklist.c

```diff
@@ -1034,7 +955,7 @@ REDIS_STATIC void _quicklistInsert(quicklistIter *iter, quicklistEntry *entry,
         new_node->entry = lpPrepend(new_node->entry, value, sz);
         new_node->count++;
         quicklistNodeUpdateSz(new_node);
-        quicklistRecompressOnly(new_node);
+        quicklistCompress(quicklist, new_node);
         quicklistRecompressOnly(node);
```

Comment:
```suggestion
        quicklistCompressNode(new_node);
        quicklistCompress(node);
```
Maybe it's more appropriate.
Since the newnode has data added to it, we'll recompress it anyway.

Reply:
No, new_node may not in the compress depth. And node->entry is not changed, node->recompress is reliable.